src/calendar-list/index.js

```diff
@@ -219,6 +219,7 @@ class CalendarList extends Component {
         initialScrollIndex={this.state.openDate ? this.getMonthIndex(this.state.openDate) : false}
         getItemLayout={this.getItemLayout}
         scrollsToTop={this.props.scrollsToTop}
+        initialNumToRender={1}
```

Comment:
can this be a prop?

Reply:
What is the usecase when `initialNumToRender` should not be 1? As far as I understand, the property is there when we are showing multiple items in the list on the page. And since in the calendar, the user can only see one month at a time, I think it should always be 1. What do you think?

Reply:
Not all time, my apps renders it more than once 
<img width="387" alt="screen shot 2018-10-04 at 18 11 30" src="https://user-images.githubusercontent.com/1761980/46490835-fde50500-c800-11e8-845e-03e7f9a4100b.png">